lib/kbsecret/cli/kbsecret-rm

```diff
@@ -12,29 +12,33 @@ cmd = CLI.create do |c|
   c.slop do |o|
     o.banner = <<~HELP
       Usage:
-        kbsecret rm [options] <record>
+        kbsecret rm [options] <record [records ...]>
     HELP
 
     o.string "-s", "--session", "the session containing the record", default: :default
     o.bool "-i", "--interactive", "ask for confirmation before deleting"
   end
 
   c.dreck do
-    string :label
+    list :string, :labels
   end
 
   c.ensure_session!
 end
 
-label = cmd.args[:label]
+records = cmd.session.records :login
 
-cmd.die "Can't delete a nonexistent record." unless cmd.session.record? label
+selected_records = records.select do |record|
+  cmd.args[:labels].include? record.label
+end
+
+cmd.die "Can't delete a nonexistent record." unless selected_records
```

Comment:
`selected_records` will always be an `Array`, so this condition will always fail.

It should probably be something like this:

```ruby
cmd.die "No such record(s)." if selected_records.empty?
```